### DISCUSSION TITLE
Is lobe chat compatible with openwebui functions ?

### DISCUSSION THREAD

**Discussion by Greatz08** [lobehub/lobehub#4920] [General]
There are some great functions which i use in openwebui so i was wondering if it is possible to use those in lobechat ? if it compatible or we can recreate those without much difficulty in lobe-chat then i would love to try out this project

**Comment by arvinxx** [comment]
I have read the openwebui's docs about functions. I think it's not a good idea to intergate openwebui functions in lobechat as our project is a nextjs based project but owu is a python project. It's difficult to have this intergation.

> There are some great functions which i use in openwebui 

If you can see what's functions you like, maybe I can tell you whether we have the same ability in lobe chat.
